File: beginner/javascript/1865.js
var input = require('fs').readFileSync('stdin', 'utf8').split('\n');

var n = input[0]

for(var i = 1; i <= n; i++){
    var entrada = input[i].split(" ");

    var nome = entrada[0];
    var numero = parseInt(entrada[1]);

    if(nome == "Thor")
        console.log("Y");
    else    
        console.log("N")
}

File: beginner/java/1865.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);
        
        int i, n, x;

        n = entrada.nextInt();
        entrada.nextLine(); 

        for(i = 0; i < n; i++){
            String partida = entrada.nextLine();
            String name[] = partida.split(" ");

            String nome = name[0];
            int numero = Integer.valueOf(name[1]);

            if(nome.equals("Thor"))
                System.out.println("Y");
            else
                System.out.println("N");
        }
    }
}
